### PR TITLE
Update notifications ui for the redesign

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
#### Summary

Update notifications ui for the redesign

#### Description

Adjusted notifications styles for them to look as expected in both the current and the new theme.

Default theme:
image

New theme:
image

### CODE REVIEW

File: packages/react-notifications/src/components/notification/notification.styles.ts [hunk 1]
```diff
@@ -60,23 +76,30 @@ const getStylesForIcon = (props: StyleProps) => css`
...
     flex-grow: 1;
-    padding: 0 ${customProperties.spacingS};
+    padding: 0
+      ${props.isNewTheme ? designTokens.spacingM : designTokens.spacingS};
     margin: 0;
     font-size: ${props.domain === NOTIFICATION_DOMAINS.SIDE
-      ? '0.929rem'
+      ? props.isNewTheme
+        ? '1rem'
+        : '0.929rem'
```
Comment (CarlosCortizasCT) [inline packages/react-notifications/src/components/notification/notification.styles.ts:102]: I wasn't sure about importing the `application-components` package in this one just for this two variants.

Please let me know what you think.

**Comment by tdeekens** [comment]
Nice, thanks. Any chance you can add a before/after screenshot to the description?

**Comment by CarlosCortizasCT** [comment]
> Nice, thanks. Any chance you can add a before/after screenshot to the description?

Done!

File: packages/react-notifications/src/components/notification/notification.styles.ts (hunk 1 above)
Comment (tdeekens) [reply packages/react-notifications/src/components/notification/notification.styles.ts:102]: In the end it's gonna be 1rem after we remove the old theme. So I think it's fine like this.

**Review by tdeekens — APPROVED** [review]

**Review by emmenko — APPROVED** [review]
Nice, thanks!

**Review by Rhotimee — APPROVED** [review]
🚀

**Review by kark — APPROVED** [review]
💯

File: packages/react-notifications/src/components/notification/notification.styles.ts [hunk 2]
```diff
@@ -81,8 +81,13 @@ const getStylesForNotificationIcon = (props: StyleProps) => css`
   background: ${getColorByType(props.type)};
 `;
 
-const getStylesForCloseIcon = (isNewTheme = false) => css`
-  ${isNewTheme ? '& svg { width: 12px; height: 12px; }' : ''}
+const getStylesForCloseIcon = (props: StyleProps) => css`
+  display: flex;
+  justify-content: center;
+  ${props.isNewTheme ? '& svg { width: 16px; height: 16px; }' : ''}
+  ${props.isNewTheme && props.domain !== NOTIFICATION_DOMAINS.SIDE
```
Comment (CarlosCortizasCT) [inline packages/react-notifications/src/components/notification/notification.styles.ts:88]: In the new theme we want the close icon to be white for `page` and `global` domain notifications.

File: packages/react-notifications/src/components/notification/notification.tsx [hunk 3]
```diff
@@ -45,15 +46,23 @@ type PropsIcon = {
...
-  if (props.type === NOTIFICATION_KINDS_SIDE.info)
-    return <InfoIcon color={props.color} />;
-  if (props.type === NOTIFICATION_KINDS_SIDE.warning)
+  }
+  if (props.type === NOTIFICATION_KINDS_SIDE.info) {
+    return props.isNewTheme ? (
+      <InformationIcon color={props.color} />
+    ) : (
+      <InfoIcon color={props.color} />
+    );
```
Comment (CarlosCortizasCT) [inline packages/react-notifications/src/components/notification/notification.tsx:61]: This icon changes from the old to the new theme.

**Comment by CarlosCortizasCT** [comment]
I updated some styles after @FilPob review:
* close icon size
* close icon color in `page` and `global` notifications (must be white)

image

**Pull request merged**